web-components/src/components/icons/EmptyCartIcon.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { SxProps } from '@mui/material';
+import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon';
+
+import { Theme } from '~/theme/muiTheme';
+
+interface IconProps extends SvgIconProps {
+  sx?: SxProps<Theme>;
+}
+
+export default function EmptyCartIcon({
+  sx = [],
+  ...props
+}: IconProps): JSX.Element {
```

Comment:
```suggestion
  export default function EmptyCartIcon(props: SvgIconProps): JSX.Element {
```
`sx` should already be in `SvgIconProps` so I think you can pass directly (should only have to manually override like this if you’re passing some `sx` defaults on line 20, which you should be able to delete in this case)

(just an FYI - no need to change before merge)

Reply:
fixed